app/services/hide_from_search_validator.rb

```diff
@@ -0,0 +1,20 @@
+class HideFromSearchValidator
+  attr_reader :base_path
+
+  def initialize(base_path)
+    @base_path = base_path
+  end
+
+  def valid?(document)
```

Comment:
Good point, though I left it out of consistency with the other validator service. I will leave for now, but if we do more of these we should probably reconsider the naming.